cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -139,13 +141,15 @@ public static void CopyTo(this IBuffer source, byte[] destination)
         {
             if (source == null) throw new ArgumentNullException(nameof(source));
             if (destination == null) throw new ArgumentNullException(nameof(destination));
+            if (source.Length == 0 && destination.Length == 0) return;
 
             CopyTo(source, 0, destination, 0, checked((int)source.Length));
         }
 
 
         public static void CopyTo(this IBuffer source, uint sourceIndex, byte[] destination, int destinationIndex, int count)
         {
+            if (source.Length == 0 && destination.Length == 0) return;
```

Comment:
You should move this after the null checks, otherwise we can get null pointer here.

Reply:
We also probably should do source.length == 0 && count == 0 rather than destination.length